statsmodels/compat/__init__.py

```diff
@@ -1 +1,24 @@
-from .python import *
```

Comment:
I find this replacement too painful to maintain, i.e. we have to change the same function name 3 times when there are changes.

Reply:
The `__all__` bit I get, but the `*` import is a pattern worth avoiding.

Reply:
> but the * import is a pattern worth avoiding.

not in `__init__` or `api.py` when it's not important.

(In exclipse, I get some false red for imports from other packages, e.g. scipy or pytest, that doesn't recognize that a function is available there. I don't know whether that's related to `*` imports.)

Reply:
> not in __init__ or api.py when it's not important.

Its easy and it helps avoid mistakes.  e.g. are you sure the `compat` namespace is the same in py2/py3?  

Of course it's not a big deal.  All of these things are individually not-big-deals.  In aggregate they make for more readable, more maintainable code.